<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music_player_window</class>
 <widget class="QMainWindow" name="music_player_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>321</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song name</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_open_folder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string>Open folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_music">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string>Open music</string>
    </property>
   </widget>
   <widget class="QSlider" name="prograss_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_previous">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mute">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>465</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_random_song_list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string>Ramdom song list</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_song">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { 
            font-size: 12px;
            font: bold large &quot;Aerial&quot;;
            color:rgb(17,17,17);
            border-width: 1px;
            border-radius: 6px;
            border-bottom-color: rgb(150,150,150);
            border-right-color: rgb(165,165,165);
            border-left-color: rgb(165,165,165);
            border-top-color: rgb(180,180,180);
            border-style: solid;
            padding: 4px;
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        
        QPushButton:hover {
            border-top-color: rgb(76,195,255);
            border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgb(76,195,255), stop:1 rgb(76,195,255));
            border-bottom-color: rgb(76,195,255);
            background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }
        QPushButton:pressed{
            background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
        }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
